src/platforms/README.md

```diff
@@ -0,0 +1,29 @@
+# Platforms and platform support files
+
+This directory contains the implementation of platforms and support file
+used by (multiple) platforms.
+
+## Implementation directories
+
+* native: Firmware for [Black Magic Probe](https://1bitsquared.com/collections/embedded-hardware/products/black-magic-probe)
```

Comment:
the link should be https://1bitsquared.com/products/black-magic-probe (it is more robust to potential category changes in the future)

Reply:
Excellent catch, wasn't sure what the best link was so that answers that for us